crates/bevy_render/src/primitives/mod.rs

```diff
@@ -81,17 +81,16 @@ impl Sphere {
     }
 }
 
-/// A plane defined by a unit normal and distance from the origin along the normal
-/// Any point `p` is in the plane if `n.p + d = 0`
-/// For planes defining half-spaces such as for frusta, if `n.p + d > 0` then `p` is on
-/// the positive side (inside) of the plane.
+/// A half-space defined by a unit normal and distance from the origin along the normal
+/// Any point `p` is considered to be within the half-space,
+/// or on the positive side (inside) of the plane, if the equation `n.p + d > 0` is satisfied.
 #[derive(Clone, Copy, Debug, Default)]
-pub struct Plane {
+pub struct HalfSpace {
     normal_d: Vec4,
 }
 
-impl Plane {
-    /// Constructs a `Plane` from a 4D vector whose first 3 components
+impl HalfSpace {
+    /// Constructs a `HalfSpace` from a 4D vector whose first 3 components
     /// are the normal and whose last component is the distance along the normal
```

Comment:
Could you do a second pass on the docs?

Before, the docs referred to `Plane` and "plane" as a flat infinite 2d surface in 3d.

`HalfSpace` isn't exactly the same as `Plane`, it's half of the 3d space, where each half is separated by the 2d surface.

All doc strings on `HalfSpace` methods should be update to reflect this semantic change. Just replacing Plane with HalfSpace is not enough.

For example, here:

> Constructs a `Plane` from a 4D vector whose first 3 components are the normal and whose last component is the distance along the normal from the origin.

This is the classical mathematical definition of a plane, it doesn't work if you replace plane with HalfSpace, because that's not what an half space is. A half space doesn't have "3 components"

A half space is defined in term of the plane that split 3d space in two. I would suggest you replace "plane" with "plane that splits the 3d space in 2" (I don't like this wording, please don't re-use it, find a better one)

See the wikipedia article: <https://en.wikipedia.org/wiki/Half-space_(geometry)>

Reply:
My apologies, I have revised the documentation in the newest commit. I appreciate your feedback.